### DISCUSSION TITLE
Variable expansion to support text in legend option for a query in a panel

### DISCUSSION THREAD

**Discussion by eleijonmarck** [grafana/grafana#66919] [Dashboards, Panels & Visualizations]
I have a am using a mixed-datasource from prometheus and I am able to get specific texts in one way but not another that works without a text added to it. it's better explained with an example:

I want the legend to display dev/ops depending on what env in the labels;
yea this works
```
{{handler}} , {{method}}
```
this works
```
{{handler, method}}
```

**Adding a text to the labels**
```
dev- {{handler}} , {{method}}
```

this does not work
```
dev- {{handler, method}}
```

This would make it more consistent

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82732. Feel free to continue the discussion around this there. Thank you!
